dart/tool/service.md

```diff
@@ -2086,13 +2086,11 @@ class Instance extends Object {
   //   Closure
   @Function closureFunction [optional];
 
-  // TODO(devoncarew): this can return an InstanceRef
```

Comment:
Ah, that's why this was returning an InstanceRef!

Reply:
Yep! It's just a strange side effect of returning a reference to the actual null instance in the VM for null in certain circumstances.